### PR TITLE
CMS Page and category

### PR DESCRIPTION
**Pull request by fouratachour**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Create, update, delete and delete with bulk actions CMS categories and CMS page 
| Type?         | new feature
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | /
| How to test?  | path=full/15_design/* npm run specific-test -- --URL=BackOfficeURL

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8900)

### CODE REVIEW

File: tests/E2E/test/selectors/BO/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = Object.assign({},
   require('./shopParameters'),
   require('./installation'),
   require('./international'),
-  require('./employee_page')
+  require('./employee_page'),
+  require('./desgin')
```
Comment (mickaelandrieu) [inline tests/E2E/test/selectors/BO/index.js]: is it desgin or design?

File: tests/E2E/test/selectors/BO/desgin/pages.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+module.exports = {
```
Comment (mickaelandrieu) [inline tests/E2E/test/selectors/BO/desgin/pages.js]: imo the folder name is wrong: should be design instead of desgin: isn't it?

**Review by mickaelandrieu — COMMENTED** [review]
Change the folder name

**Comment by fouratachour** [comment]
Hello @mickaelandrieu 
Done :+1: 
Thanks for your review.

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Thanks @fouratachour! :)

**Pull request merged**